src/elements/post-card/post-card.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import {
+  Hero,
+  HeroProps,
+  Premiere,
+  PremiereProps,
+  Item,
+  ItemProps,
+  Quote,
+  QuoteProps,
+  Default,
+  DefaultProps
+} from './variants'
+import { TimeAgoProps } from '@jesus-film/ark.elements.time-ago'
+
+export type PostCardProps = HeroProps | PremiereProps | ItemProps | QuoteProps & TimeAgoProps | DefaultProps & TimeAgoProps
```

Comment:
the parent shouldn't be concerned with modifying the underlying types.
```suggestion
export type PostCardProps = HeroProps | PremiereProps | ItemProps | QuoteProps | DefaultProps
```